src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -26,6 +26,10 @@ public SchedulerConf() {
     setConf(configuration);
   }
 
+  public String getExecutorPath() {
+    return getConf().get("mesos.hdfs.executor.config.path", "..");
```

Comment:
Is `..` a valid/reasonable executor path? Is that the default when untarring our build?